src/main/java/Storage.java

```diff
@@ -0,0 +1,85 @@
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.Path;
+import java.util.List;
+
+/**
+ * Class that sets up the save/load system in project Duke.
+ */
+
+public class Storage {
+
+    static Path dir = Paths.get("data");
+    static Path file = Paths.get("data/saveFile.txt");
+
+    public Storage() {
+        if (!Files.isDirectory(dir)) {
+            try {
+                Files.createDirectory(dir);
+                if (!Files.exists(file)) {
+                    Files.createFile(file);
+                }
+            } catch (IOException E) {
+                System.out.println(E.getMessage() + " IO Exception detected.");
+            }
+        }
+    }
+
+    /**
+     * Saves the tasklist in /data/saveFile.txt
+     * @param store the current tasklist of the user
+     */
+    public void saveFile(TaskList store) {
+        try {
+            String save = "";
+            for (int i = 0; i < store.storage.size(); i++) {
+                save += store.storage.get(i).getData() + "\n";
+            }
+            save = save.replaceAll("\u2713", "1");
+            save = save.replaceAll("\u2718", "0");
+            save = save.replaceAll("\\[", "" );
+            save = save.replaceAll("]", "");
+            save = save.replaceAll("\\(", "");
+            save = save.replaceAll("\\)", "");
```

Comment:
Do you thinking creating a utility function to re-format String 'save' at one go will be helpful? It may make the method shorter and more concise:)